packages/office-addin-debugging/src/port.ts

```diff
@@ -75,7 +75,7 @@ export function getProcessIdsForPort(port: number): Promise<number[]> {
           lines.forEach((line) => {
             /* eslint-disable-next-line @typescript-eslint/no-unused-vars */
             const [protocol, localAddress, foreignAddress, status, processId] = line.trim().split(" ").filter((text) => text);
-            if (processId !== undefined && processId !== '0') {
+            if (processId !== undefined && processId.trim() !== '0') {
```

Comment:
Would it be better to use .trim() within the .filter() so processId another other text components are all trimmed? That would ensure the trailing \r is removed from the processId string and not used later outside of the check for zero.

`.filter((text) => text.trim())`

Reply:
Since 3 of the 5 elements of the array aren't used anyway and `parsePort()` processes `localAddress`  with a regex, I'd keep the change as minimal as possible